packages/app/src/cli/services/app/config/link.ts

```diff
@@ -148,6 +151,10 @@ export function mergeAppConfiguration(
     },
   }
 
+  // Get remote app configuration from app modules
+  const remoteAppConfig = await getRemoteAppConfig(remoteApp.apiKey)
```

Comment:
I'm not sure I like having a query inside the `mergeConfig` function 🤔 
Wouldn't it be cleaner to get the config before and pass it? so that we can keep this function pure.

Reply:
I'm good with that change